File: SitioVentas.Entities/Entities/Foto.cs
using Dapper.Contrib.Extensions;

namespace SitioVentas.Entities.Entities
{
    [Table("foto")]
    public class Foto
    {
        [Key]
        public int Id { get; set; }
        public string Nombre { get; set; }
        public string Ruta { get; set; }
        public string Tipo { get; set; }
        public int Prioridad { get; set; }
        public int ItemId { get; set; }
    }
}

File: SitioVentas/ClientApp/src/app/demo/api/foto.ts
import { SafeUrl } from "@angular/platform-browser";
import { IFoto } from "./foto.interface";

export class Foto implements IFoto{
    id!: number;
    nombre!: string;
    ruta!: string;
    tipo!: string;
    prioridad!: number;
    archivo!: string;
    itemId!: number;
    imageUrl!: SafeUrl;
}
